lib/index.js

```diff
@@ -52,6 +52,9 @@ function exec(command, ...args) {
   return deferred.promise;
 }
 
+const endsWith = (suffix, str) =>
+  str.indexOf(suffix, str.length - suffix.length) !== -1;
+
```

Comment:
We loaded core-js globally, causing every string to be instantiated by the custom core-js polyfill. Now we don't pollute the global namespace and don't have these functions anymore.